hedera-node/test-clients/README.md

```diff
@@ -1,249 +1,282 @@
 
 ### Embedded networks
 
-We can address the testing deficiencies of a subprocess network by trading off the realism of a live network for
-greater control. Maximum control is achieved by directly instantiating a `Hedera` instance in the test process, and
-initializing it with fake `Platform`, `ServicesRegistry`, and `HederaState` implementations that are created
-and managed by the testing framework. Instead of submitting transactions through gRPC, we can call the ingest,
-query, pre-handle, and handle workflows directly.
-
-At this point, the main tradeoff is between supporting concurrent test execution and achieving repeatable
-results. The `ConcurrentEmbeddedNetwork` supports concurrent test execution, but the consensus time is still tied
-to real time. The `RepeatableEmbeddedNetwork` achieves repeatable results by instead using virtual consensus time;
-but this requires single-threaded execution and exclusive use of Ed25519 keys, since the ECDSA signing process has
-inherent randomness.
-
-Either way, we can now directly access and manipulate the network's internal state; and we can submit transactions
-that would be rejected by a real network. Furthermore, in repeatable mode, `SpecOperation`'s that would normally
-sleep in real time switch to immediately advancing virtual time the requested amount. This lets us test staking
-rewards, contract expiration, and other time-dependent behaviors orders of magnitude faster than we could otherwise.
+We can address the testing deficiencies of a subprocess network by trading off the realism of a live
+network for greater control. Maximum control is achieved by directly instantiating a `Hedera`
+instance in the test process, and initializing it with fake `Platform`, `ServicesRegistry`, and
+`HederaState` implementations that are created and managed by the testing framework. Instead of
+submitting transactions through gRPC, we can call the ingest, query, pre-handle, and handle
+workflows directly.
+
+At this point, the main tradeoff is between supporting concurrent test execution and achieving
+repeatable results. The `ConcurrentEmbeddedNetwork` supports concurrent test execution, but the
+consensus time is still tied to real time. The `RepeatableEmbeddedNetwork` achieves repeatable
+results by instead using virtual consensus time; but this requires single-threaded execution and
+exclusive use of Ed25519 keys, since the ECDSA signing process has inherent randomness.
+
+Either way, we can now directly access and manipulate the network's internal state; and we can
+submit transactions that would be rejected by a real network. Furthermore, in repeatable mode,
+`SpecOperation`'s that would normally sleep in real time switch to immediately advancing virtual
+time the requested amount. This lets us test staking rewards, contract expiration, and other
+time-dependent behaviors orders of magnitude faster than we could otherwise.
 
 <img src="./docs/assets/embedded-network.png" width="900">
```

Comment:
Do we need a set width here?

If not I'd convert the image to markdown syntax.

Reply:
Updated all the images to use markdown syntax. Thanks @rbarkerSL 😎